shaders/src/ambient_occlusion.fs

```diff
@@ -17,6 +17,7 @@ float unpack(vec2 depth) {
 }
 
 float evaluateSSAO() {
+#if defined(BLEND_MODE_OPAQUE) || defined(BLEND_MODE_MASKED)
```

Comment:
This is still something I'd like to think about. It seems correct but it bothers me.

Reply:
In what way? is it the implementation here, or just the idea of not applying SSAO to objets that are not in the depth buffer?